packages/mobile/src/components/notifications/content/UserImages.tsx

```diff
@@ -37,68 +36,77 @@ const styles = StyleSheet.create({
 type UserImagesProps = {
   notification: Notification
   users: User[]
-  onGoToRoute: (route: string) => void
 }
 
-const getImageURI = (user: any) => {
-  let node: string
-  if (user.creator_node_endpoint) {
-    node = user.creator_node_endpoint.split(',')[0]
-  } else {
-    node = USER_METADATA_NODE
-  }
-  if (user.profile_picture) {
-    return `${node}/ipfs/${user.profile_picture}`
-  }
-  if (user.profile_picture_sizes) {
-    return `${node}/ipfs/${user.profile_picture_sizes}/150x150.jpg`
-  }
-  return null
-}
+const UserImage = ({
+  user,
+  allowPress = true
+}: {
+  user: User
+  allowPress?: boolean
+}) => {
+  const dispatch = useDispatch()
+  const navigation = useNavigation()
+
+  const handlePress = useCallback(() => {
+    navigation.navigate({
+      native: { screen: 'Profile', params: { handle: user.handle } },
+      web: { route: getUserRoute(user), fromPage: NOTIFICATION_PAGE }
+    })
+    dispatch(close())
+  }, [navigation, user, dispatch])
+
+  const profilePicture = useUserProfilePicture(
+    user?.user_id,
+    user?._profile_picture_sizes,
+    SquareSizes.SIZE_150_BY_150
+  )
 
-const UserImage = ({ source }: { source: ImageSourcePropType }) => {
   const imageStyle = useTheme(styles.image, {
     backgroundColor: 'neutralLight4'
   })
-  const [didError, setDidError] = useState(false)
   return (
-    <Image
-      style={imageStyle}
-      source={
-        didError
-          ? require('app/assets/images/imageProfilePicEmpty2X.png')
-          : source
-      }
-      // TODO: Gracefully handle error and select secondary node
-      onError={() => setDidError(true)}
-    />
+    <TouchableOpacity
+      activeOpacity={0.7}
+      onPress={allowPress ? handlePress : undefined}
+      key={user.user_id}
+    >
+      <Image style={imageStyle} source={{ uri: profilePicture }} />
```

Comment:
Should we use DynamicImage here?

Reply:
It's a static size and the smallest we have, so probably ok? what do u think

Reply:
We would get a nice fade in with it! I'm fine with either